slides/index.html

```diff
@@ -440,14 +449,13 @@ <h1 style="color:DeepSkyBlue">Docker</h1>
           <h3>Images</h3>
           <pre><code class="bash" data-trim data-noescape>
 $ mkdir -p workshop/ex1
+> mkdir -path workshop/ex1
 
 <span class="fragment">$ cd workshop/ex1</span>
 
 <span class="fragment">$ touch Dockerfile</span>
 
 <span class="fragment">$ touch entrypoint.sh</span>
-
-<span class="fragment">$ chmod u+x entrypoint.sh</span>
```

Comment:
Windows doesn't have an equivalent to this. I moved the command into the Dockerfile below.